CHANGELOG.md

```diff
@@ -73,6 +73,7 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Added `OnClientOwnershipGained` and `OnClientOwnershipLost` events on Actors and ActorComponents. These events trigger when an Actor is added to or removed from the ownership hierarchy of a client's PlayerController.
 - You can now generate valid schema for classes that start with a leading digit. The generated schema class will be prefixed with `ZZ` internally.
 - Handover properties will be automatically replicated when required for load balancing. `bEnableHandover` is off by default.
+- Added `bWorkerFlushAfterRPC` (defaulted false) which publishes changes to the GDK worker queue faster to allow for lower latencies. 
```

Comment:
```suggestion
- Added `bWorkerFlushAfterRPC` (defaulted false) which publishes changes to the GDK worker queue after every RPC call to allow for lower latencies. 
```

Reply:
?

Reply:
Resolved this by accident, removed the other options!